force-app/main/domain/LegacyHouseholds.cls

```diff
@@ -295,8 +317,10 @@ public inherited sharing class Households extends fflib_SObjects {
         updateOneToOneAccounts(
                 contactsInstance.getOneToOneContactsToUpdate(), dmlWrapper);
 
-        HouseholdAccounts.handleContactsAfterUpdate(contactsInstance, dmlWrapper);
-        HouseholdAccounts.renameHHAccounts(
+        List<Id> householdIdsToUpdate =
+                contactService.moveOppsForContactsSwitchingAccounts(contactsInstance);
+        householdService.updateHouseholds(householdIdsToUpdate, dmlWrapper);
+        Households.updateNameAndMemberCount(
```

Comment:
There are a lot of references here to the Households account.  That doesn't feel right.  Is it an interim step??

Reply:
Yes, this LegacyHouseholds class needs an internal refactor like we are currently doing for Households.  Lots of renaming to do and definitely some duplicate logic.  The references between these classes need to be decoupled also.  Need to pull some logic out into shared HouseholdService.